.github/workflows/azure_e2e_test.yml

```diff
@@ -1,40 +1,16 @@
 name: 'Azure E2E test'
-on:
+'on':
+  schedule:
+    # run twice a day at 10:00 and 22:00 UTC every day of the week
+    - cron: "0 10/12 * * 1-5"
   workflow_dispatch:
-    inputs:
-      target-branch:
-        description: 'Branch of CBAS to run tests on'
-        required: true
-        default: 'develop'
-        type: string
-      # Replace user data with seeded data provided by devOps (once available)
-      owner-subject:
-        description: 'Owner of billing project'
-        required: true
-        default: 'hermione.owner@quality.firecloud.org'
-        type: string
-      service-account:
-        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials'
-        required: true
-        default: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
-        type: string
```

Comment:
Am I reading this right, that manual triggering is still possible, it just always generates all of the parameters itself now, rather than needing us to supply them?

Reply:
Yes that is correct. These values are static and had to be setup in tfvars file in terraform-ap-deployments repo. Hence I removed them from here. The manual triggering of workflow is still possible.